src/ape_console/_cli.py

```diff
@@ -24,6 +32,57 @@ def cli(cli_ctx, network):
     return console(verbose=verbose)
 
 
+def import_extras_file(file_path) -> ModuleType:
+    """Import a module"""
+    loader = SourceFileLoader(file_path.name[:-3], str(file_path))
+    spec = spec_from_loader(loader.name, loader)
+
+    assert spec is not None
+
+    module = module_from_spec(spec)
+    loader.exec_module(module)
+
+    return module
+
+
+def load_console_extras(namespace: dict[str, Any]) -> dict[str, Any]:
+    """load and return namespace updates from ape_console_extras.py  files if
+    they exist"""
+    global_extras = config.DATA_FOLDER.joinpath(CONSOLE_EXTRAS_FILENAME)
+    project_extras = config.PROJECT_FOLDER.joinpath(CONSOLE_EXTRAS_FILENAME)
+
+    for extras_file in [global_extras, project_extras]:
+        if extras_file.is_file():
+            module = import_extras_file(extras_file)
+            init_extras = getattr(module, "init_extras", None)
+
+            # If found, execute and init_extras() function.
+            if init_extras is not None:
+                # Figure out the kwargs the func is looking for and assemble
+                # from the original namespace
+                func_spec = inspect.getfullargspec(init_extras)
+                init_kwargs: Dict[str, Any] = {k: namespace.get(k) for k in func_spec.args}
+
+                # Execute functionality with existing console namespace as
+                # kwargs.
+                extras = init_extras(**init_kwargs)
+
+                # If init_extras returned a dict expect it to be new symbols
+                if type(extras) == dict:
```

Comment:
```suggestion
                if isinstance(extras, dict):
```

This will make it so subclasses of `dict` are also supported